src/main/resources/static/javascript/mainApp/general.controller.js

```diff
@@ -1412,8 +1403,7 @@
                     listName: "owners"
                 });
             } else {
-                const userType = "owner";
-                $scope.createRemoveErrorModal(userType);
+                $scope.launchDynamicModal(Message.Title.LAST_OWNER_NOT_REMOVED, Message.Body.LAST_OWNER_NOT_REMOVED);
```

Comment:
I believe we need a Modal controller this Dynamic modal is being used across multiple controllers making it difficult to write tests for.